CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.0.0-beta.15 (2017-03-22)
+
+- Images! Images stored in `src/assets` and referenced via `~/assets/` in all HTML/SCSS files are now bundled.
+- Added `preload` and `postload` plugin hooks.  Beginning initial work for a lifecycle hooks. (Thanks @Blackbaud-SteveBrush!)
```

Comment:
I'd change "Beginning" to "Started" to be consistent with the past tense verb in the previous sentence. ... Also, I think we should delete "a" before "lifecycle hooks" since it is plural. "Either that or change "hooks" to "hook.")